pallets/pools-system/src/migrations/mod.rs

```diff
@@ -89,9 +97,18 @@ pub mod altair {
 		pub reserve: ReserveDetails<Balance>,
 	}
 
-	#[frame_support::storage_alias]
-	type Pool<T: Config> = StorageMap<
-		Pallet<T>,
+	/// The old prefix we used when using pallet-pools as
+	/// `Pools` in the construct_runtime!-macro
+	pub struct PoolPrefix;
```

Comment:
I know this is temporary code, but we can name this code more clearly to not rely as much on the comments

```suggestion
	pub struct OldPoolsPrefix;
```